.github/workflows/pr-realm-js.yml

```diff
@@ -171,7 +198,7 @@ jobs:
         with:
           name: realm-js-prebuilds
           path: |
-            dependencies.list
+            README.md
```

Comment:
I think the dependencies.list will go away soon, so I don't want the upload to rely on its existence.